test/test_suite_test.go

```diff
@@ -184,8 +184,11 @@ var _ = BeforeAll(func() {
 		return
 	}
 
-	if progressChan := goReportVagrantStatus(); progressChan != nil {
-		defer func() { progressChan <- err == nil }()
+	if config.CiliumTestConfig.Kubeconfig == "" {
```

Comment:
This is not a good way of figuring out whether we are running on vagrant, until lately we have been running tests on vagrant cluster with defined kubectl in CI and we will likely get back to it.

Since we are doing some stuff during setup even if we are not using Vagrant, I think we should change `goReportVagrantStatus` into `goReportSetupStatus` and change `Spinning up[ vagrant VMs...` text into something like `Setting up test suite...` or something similar.

Reply:
fixed